medcat/utils/meta_cat/ml_utils.py

```diff
@@ -110,8 +113,8 @@ def predict(model: nn.Module, data: List[Tuple[List[int], int, Optional[int]]],
     return predictions, confidences
 
 
-def split_list_train_test(data: List, test_size: float, shuffle: bool = True) -> Tuple:
-    """Shuffle and randomply split data
+def split_list_train_test(data: List, test_size: int, shuffle: bool = True) -> Tuple:
```

Comment:
method sig has now been changed, just use a new param here, test_size_int: int, and keep the float which is a % size for test. If both are set raise a ValueError( 'test_size and test_size_int both set only use one' )

Reply:
Done!